roles/sensu/server/templates/redis.json.j2

```diff
@@ -1,6 +1,6 @@
  {
    "redis": {
-     "port": {{ redis_listen_port }},
-     "host": "{{ sensu_server_interface }}" 
+     "port": {{ sensu_redis_port }},
+     "host": "{{ sensu_redis_host }}"
```

Comment:
Either `sensu_rabbitmq_server` and `sensu_redis_server`, or `sensu_rabbitmq_host` and `sensu_redis_host`, but don't mix and match :)